modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/diagnostic/pagelocktracker/PageLockTrackerManager.java

```diff
@@ -105,7 +104,8 @@ public PageLockTrackerManager(IgniteLogger log) {
      * Default constructor.
      */
     public PageLockTrackerManager(IgniteLogger log, String managerNameId) {
-        this.trackingEnabled = getInteger(IGNITE_PAGE_LOCK_TRACKER_TYPE, HEAP_LOG) != -1;
+        // Only enable tracking if tracking is explicitly requested.
+        this.trackingEnabled = getInteger(IGNITE_PAGE_LOCK_TRACKER_TYPE, -1) != -1;
```

Comment:
Could you please change the default value in `IgniteSystemProperties` .
See https://github.com/gridgain/gridgain/blob/master/modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java#L1934